JumpscaleBuildersCommunity/network/BuilderGeoDns.py

```diff
@@ -1,7 +1,8 @@
 from Jumpscale import j
+from JumpscaleBuilders.runtimes.BuilderGolangTools import BuilderGolangTools
 
 
-class BuilderGeoDns(j.baseclasses.builder):
+class BuilderGeoDns(BuilderGolangTools):
```

Comment:
shouldn't this be builderbaseclass?

Reply:
BuilderGoLangTools uses j.classes.builder, but it is needed for the profile env set. So instead of using in the builder `j.builders.runtimes.go.get(....)` we use `self.get(...)`